docs/API.md

```diff
@@ -471,6 +722,100 @@ await realm.getWorldDownload()
 
 ---
 
+#### getSubscriptionInfo
+
+() => Promise\<RealmSubscriptionInfo>
+
+Gets the basic subscription
+
+```js
+await realm.getSubscriptionInfo()
+```
+
+<details>
+<summary>Output</summary>
+
+```ts
+{
+  startDate: number
+  daysLeft: number
+  subscriptionType: string
+}
+```
+
+</details>
+
+---
+
+#### getSubscriptionInfoDetailed
+
+() => Promise\<RealmSubscriptionInfoDetailed>
+
+Gets the detailed subscription info
+
+```js
+await realm.getSubscriptionInfoDetailed()
```

Comment:
Why do we need two functions getSubscriptionInfo for this? Why not just an extra param to add more data?